recipes/openssl/3.x.x/conandata.yml

```diff
@@ -1,10 +1,10 @@
 sources:
+  3.0.7:
+    url: https://www.openssl.org/source/openssl-3.0.7.tar.gz
+    sha256: 83049d042a260e696f62406ac5c08bf706fd84383f945cf21bd61e9ed95c396e
   3.0.5:
     url: https://www.openssl.org/source/openssl-3.0.5.tar.gz
     sha256: aa7d8d9bef71ad6525c55ba11e5f4397889ce49c2c9349dcea6d3e4f0b024a7a
   3.0.4:
     url: https://www.openssl.org/source/openssl-3.0.4.tar.gz
     sha256: 2831843e9a668a0ab478e7020ad63d2d65e51f72977472dc73efcefbafc0c00f
-  3.0.3:
```

Comment:
Why did you removed 3.0.3?

Reply:
We remove older patch versions from all recipes when the number of releases grow.

Particular in the case of something security relevant as OpenSSL, there is absolute no good reason to use an older patch version.